data/human/human missions.txt

```diff
@@ -5428,3 +5428,71 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+mission "Quicksilver Mixup [2]"
+	name "Misposted mail delivery"
+	description "Carry misposted mail from Silver to Quicksilver by <date> and then come back with the return shipment"
+	minor
+	deadline 10 1
+	source "Silver"
+	destination "Quicksilver"
+	cargo "Misposted mail" 38
```

Comment:
```suggestion
	cargo "misposted mail" 38
```
I'm not sure if this is necessary, but generally they're in lower case.

Reply:
Necessary or not, if they're generally in lower case, i see no reason not to!